ONSdigital/census-rm-toolbox
# Minor improvements to scripts

# Motivation and Context
Scripts could be slightly improved.

# What has changed
Minor changes.

# How to test?
Cursory check to see that toolbox still working, e.g. use the write mode in both DBs.

# Links
Trello: www.trello.com
